### ISSUE TITLE
[Ollama AI] Add Server didn't work

### ISSUE THREAD

**Issue by spaceoi** [raycast/extensions#12746]
### Extension

https://www.raycast.com/massimiliano_pasquini/raycast-ollama

### Raycast Version

1.75.2

### macOS Version

14.4.1

### Description

1. Add Server didn't work. It's hard to find the add server action, and the label of server url is model. Also it's not working for remote server, even not working for local server(127.0.0.1:11434)
2. The old custom commands are not working at all. I didn't check the source code and am not sure if it's caused by the first one.

### Steps To Reproduce

Upgraded automatically from raycast store.

**Comment by MassimilianoPasquini97** [comment]
Hi @spaceoi thanks for opening this issue,

1. What do you mean for "Add Server didn't work"? Can you show the error? Are you using the right format? Server need to be added on this format "http://domain:port/" otherwise it did't let you go throw. You also mentioned that it's hard to find, can you propose a better position?
2. How mentioned on CHANGELOG.md all those changes break the old configured custom command so need to be redone.

**Comment by spaceoi** [comment]
> Hi @spaceoi thanks for opening this issue,
> 
> 1. What do you mean for "Add Server didn't work"? Can you show the error? Are you using the right format? Server need to be added on this format "http://domain:port/" otherwise it did't let you go throw. You also mentioned that it's hard to find, can you propose a better position?
> 2. How mentioned on CHANGELOG.md all those changes break the old configured custom command so need to be redone.

Hello, @MassimilianoPasquini97 thanks for your answer.

Yes the format is correct(btw, your format in replying is not correct with a redundant "/"). I am using the same remote url for "continue.dev" and open-webui.

I think Servers is one level higher than Models. This simplest modification might be update from "Manage Models" to "Manage Servers and Models".

**Comment by spaceoi** [comment]
@MassimilianoPasquini97 More precisely, if I add a new server with my another local IP(not "http://127.0.0.1:11434"), the "add server" action also throw Errors.OllamaNotInstalledOrRunning

**Comment by MassimilianoPasquini97** [comment]
> @MassimilianoPasquini97 More precisely, if I add a new server with my another local IP(not "http://127.0.0.1:11434"), the "add server" action also throw Errors.OllamaNotInstalledOrRunning

I have tried this now and it work without modifying the default url

**Comment by MassimilianoPasquini97** [comment]
> > Hi @spaceoi thanks for opening this issue,
> > 
> > 1. What do you mean for "Add Server didn't work"? Can you show the error? Are you using the right format? Server need to be added on this format "http://domain:port/" otherwise it did't let you go throw. You also mentioned that it's hard to find, can you propose a better position?
> > 2. How mentioned on CHANGELOG.md all those changes break the old configured custom command so need to be redone.
> 
> Hello, @MassimilianoPasquini97 thanks for your answer.
> 
> Yes the format is correct(btw, your format in replying is not correct with a redundant "/"). I am using the same remote url for "continue.dev" and open-webui.
> 
> I think Servers is one level higher than Models. This simplest modification might be update from "Manage Models" to "Manage Servers and Models".

Can you give me a screenshot with the form compiled with the url where you receive the error?

**Comment by MassimilianoPasquini97** [comment]
@raycastbot assign me

**Comment by spaceoi** [comment]
<img width="414" alt="" src="https://github.com/raycast/extensions/assets/3646270/2249cec7-a053-4390-a379-8a5f66d02a35">

**Comment by MassimilianoPasquini97** [comment]
Are you using the latest Ollama Server Version?

> <img alt="1" width="414" src="https://private-user-images.githubusercontent.com/3646270/336319404-2249cec7-a053-4390-a379-8a5f66d02a35.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODI1OTMsIm5iZiI6MTcxNzQ4MjI5MywicGF0aCI6Ii8zNjQ2MjcwLzMzNjMxOTQwNC0yMjQ5Y2VjNy1hMDUzLTQzOTAtYTM3OS04YTVmNjZkMDJhMzUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMDYyNDUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2ZjNzljZjU5NmE3MjkwNGI1NWUxNTBjNjFkZjFmYTFiMGJhNmE2NjUwNjUyMjllYmQxZDc3MTU5ZDRmZjlkNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.SjlvMDxzNC1pZVHItUPZYMo7Atm1zOPhug9R6GMrHW0"> <img alt="2" width="542" src="https://private-user-images.githubusercontent.com/3646270/336319865-ed95d984-8ba9-4503-afa9-1b32c55f03e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODI1OTMsIm5iZiI6MTcxNzQ4MjI5MywicGF0aCI6Ii8zNjQ2MjcwLzMzNjMxOTg2NS1lZDk1ZDk4NC04YmE5LTQ1MDMtYWZhOS0xYjMyYzU1ZjAzZTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMDYyNDUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjU3YzQzYmNmMGRkYTllZjFmMDQwNDA0MjBlYzE0MDdjNDIwZmFkZjlhMDExMjdlNzdiZmJlODg1ZGJiYTUyMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.NNowv7Kzdl9HZ-mxAQJEOePi7twwB8WbCQQbkv1qFdk"> <img alt="3" width="1089" src="https://private-user-images.githubusercontent.com/3646270/336320786-9c39be2f-be23-4ed1-aa82-9d5d8c1dd340.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODI1OTMsIm5iZiI6MTcxNzQ4MjI5MywicGF0aCI6Ii8zNjQ2MjcwLzMzNjMyMDc4Ni05YzM5YmUyZi1iZTIzLTRlZDEtYWE4Mi05ZDVkOGMxZGQzNDAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMDYyNDUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2Y4YTdlMzVjNjc2YjM4MDg5MGVhODY3YzIyZTE3NWI0NzA2Nzk2NTQ5MTA0NDU2MTZjNGM3NWFkYzNkYjZlNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.UR8YkWO9pE2bb5Gn7OAIKTjBi03k3AxkhtuSoJDydIs">

**Comment by spaceoi** [comment]
No, it's 0.1.37

**Comment by jettw** [comment]
After updating, I can't even find the add server option. It was working perfectly. I have a remote server that I connect to via Tailscale.

**Comment by MassimilianoPasquini97** [comment]
> After updating, I can't even find the add server option. It was working perfectly. I have a remote server that I connect to via Tailscale.

Reading README.md can help to solve this issue 😅. On "Manage Models" command use `CMD+K` to show available Actions and use "Add Server"

**Comment by elrrrrrrr** [comment]
![image](https://github.com/user-attachments/assets/8bd1b380-df35-45f8-a28d-08df38b5bc61)


![image](https://github.com/user-attachments/assets/909131ff-a6ba-4967-88df-927531c50468)

When adding a server, I also encountered some issues.
After pulling the model, the interface indicated that the model was successfully added.
However, when I used other actions, the interface started showing errors.
I am using the latest version of Ollama, ollama version is `0.1.48`.
